docker-compose-rails/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+services:
+  db:
```

Comment:
Would you be up for adding line-by-line comments/instructions to the docker compose file? I went with this approach [here](https://github.com/teamairship/docker-examples/blob/main/example-php-apache/docker-compose.yml) and it seemed to be good for learning purposes. Let me know what you think!